app/controllers/draft_controller.rb

```diff
@@ -10,6 +10,14 @@ def show
 
   def update
     Draft.set(current_user, params[:draft_key], params[:sequence].to_i, params[:data])
+
+    if params[:post_id] && params[:original_text]
+      post = Post.find_by(id: params[:post_id])
+      if post && post.raw != params[:original_text]
+        return render json: { success: 'OK', edit_conflict: true }
```

Comment:
I think other places in the codebase do this like
```suggestion
        return render json: success_json.merge(edit_conflict: true)
```
The difference is negligible though